MAINTAINERS.MD

```diff
@@ -1,3 +1,6 @@
 | Org                    | Name                                                |
 | -----------------------| ----------------------------------------------------|
 | Huawei | Shuting Qing |
```

Comment:
@ShutingQing Please confirm with you approval of the PR that you seeing yourself still as an active Maintainer of this API repository.

```suggestion
| Huawei | Shuting Qing | ShutingQing |
```